interhost.py

```diff
@@ -204,6 +204,60 @@ def parser_snpEff(parser=argparse.ArgumentParser()):
 __commands__.append(('snpEff', parser_snpEff))
 
 
+# =======================
+# ***  align_mafft  ***
+# =======================
+
+def parser_align_mafft(parser=argparse.ArgumentParser()):
+    parser.add_argument('inFastas', nargs='+',
+        help='Input FASTA files.')
+    parser.add_argument('outFile',
+        help='Output file containing alignment result (default format: FASTA)')
+    parser.add_argument('--localpair', default=None, action='store_true',
+        help='All pairwise alignments are computed with the Smith-Waterman algorithm.')
+    parser.add_argument('--globalpair', default=None, action='store_true',
+        help='All pairwise alignments are computed with the Needleman-Wunsch algorithm.')
+    parser.add_argument('--preservecase', default=None, action='store_true',
+        help='Preserve base or aa case, as well as symbols.')
+    parser.add_argument('--reorder', default=None, action='store_true',
+        help='Output is ordered aligned rather than in the order of the input (default: %(default)s).')
+    parser.add_argument('--gapOpeningPenalty', default=1.53,
+        help='Gap opening penalty (default: %(default)s).')
+    parser.add_argument('--ep',
+        help='Offset (works like gap extension penalty).')
+    parser.add_argument('--verbose', default=False, action='store_true',
+        help='Full output (default: %(default)s).')
+    parser.add_argument('--outputAsClustal', default=None, action='store_true',
+        help='Write output file in Clustal format rather than FASTA')
+    parser.add_argument('--maxiters', default = 0,
+        help='Maximum number of refinement iterations (default: %(default)s).')
```

Comment:
If you run MAFFT directly, it specifies in its help that the default is zero, and in the docs suggests a default of 1000 for localpair or globalpair alignment (hence why that value is used as default in that case). I'll add a note.
--maxiterate # : Maximum number of iterative refinement, default: 0